classes/submission/EditDecisionDAO.inc.php

```diff
@@ -28,13 +28,14 @@ class EditDecisionDAO extends DAO {
 	 */
 	function updateEditorDecision($submissionId, $editorDecision, $stageId = null, $reviewRound = null) {
 		if ($editorDecision['editDecisionId'] == null) {
-			$this->update(
-				sprintf(
-					'INSERT INTO edit_decisions
+			$sqlUpdate = sprintf(
+				'INSERT INTO edit_decisions
```

Comment:
Is this change doing anything? It looks like it's just reformatting.

Reply:
It's actually doing nothing special. I maybe thought for a moment it was to fix another UI freezing problem. Probably forgot to change it back after some corrections.